scripts/start-rc.bat

```diff
@@ -78,6 +78,9 @@ if exist "hazelcast-enterprise-%HAZELCAST_ENTERPRISE_VERSION%-tests.jar" (
 )
 
 
+REM necessary arguments for Java 9+
+SET JAVA_OPTS=%JAVA_OPTS%%=%--add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED
```

Comment:
why `%=%`?

Reply:
> why `%=%`?

It's what we do in `hazelcast-mono`:
https://github.com/hazelcast/hazelcast-mono/blob/55db23c82dba0dcd441824e4ba27b806f8a33eee/hazelcast/distribution/src/bin-regular/common.bat#L51

Reply:
I’d rather do it as it’s done elsewhere for consistency. I think your way makes more sense but there’s probably a reason?

Reply:
may be there is a reason in the other file but not here I believe
As indicated, the file also [sets](https://github.com/hazelcast/hazelcast-mono/blob/55db23c82dba0dcd441824e4ba27b806f8a33eee/hazelcast/distribution/src/bin-regular/common.bat#L70) it without `%=%` and google doesn't show this trickery 

I tried it locally and doesn't work i.e. doesn't replace any `%` and, in fact, removes everything whatever is previously set in `JAVA_OPTS`. So best to remove but this works though:

```
@echo off
@SET JAVA_OPTS=
@SET JAVA_OPTS=--my-opt % % %
@SET JAVA_OPTS=%JAVA_OPTS:^%=%--add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED
@echo %JAVA_OPTS%
@SET JAVA_OPTS=
```

Reply:
Actually it looks like it works ok without this at all after all

ac9f57f1ee301f5a336c36b58a8151b86bd12388

I _guess_ the args are required for Hazelcast server, but the controller - which basically just manages it's lifecycle - doesn't actually need them.

Reply:
Even better. thanks